backend/penndata/models.py

```diff
@@ -58,3 +58,14 @@ class AnalyticsEvent(models.Model):
 
     def __str__(self):
         return f"{self.cell_type}-{self.user.username}"
+
+
+class CalendarEvent(models.Model):
+    event = models.CharField(max_length=255)
+    date = models.CharField(max_length=50, null=True, blank=True)
+    # NOTE: This is bad practice, though is necessary for the time being
+    # since frontends use the string date field
+    date_obj = models.DateTimeField(null=True, blank=True)
```

Comment:
We probably need to change this in the future.

Reply:
Should have fields for a start date and a nullable end date bc some calendar events are ranges.

Reply:
Thanks for the reminder. I will leave this for you to do in a future events PR you create so we can group relevant code